index.html

```diff
@@ -34,13 +35,42 @@ <h1>nav</h1>
               About us
             </a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-              <div class="dropdown-item__section">
-                <span class="dropdown-item--title">Overview</span>
+              <!-- @matt:
+
+              1. Title
+              title shouldn't be a Modifier of dropdown-item, because it doesn't modify dropdown-item.
```

Comment:
Title does modify item, because its using slightly different css styles. It's modifying the dropdown-item 

SEE : 
Modifier is an extra class name which you add to a block/element DOM node. Add modifier classes only to blocks/elements they modify, and keep the original class: